README.md

```diff
@@ -77,9 +88,9 @@ brew link --force libpq
 git clone --recursive https://github.com/neondatabase/neon.git
 cd neon
 
-# The preferred and default is to make a debug build. This will create a 
+# The preferred and default is to make a debug build. This will create a
 # demonstrably slower build than a release build. If you want to use a release
-# build, utilize "BUILD_TYPE=release make -j`nproc`" 
+# build, utilize "BUILD_TYPE=release make -j`nproc`"
```

Comment:
While we're at it, I'd suggest:
```
# The preferred and default is to make a debug build. This will create a
# demonstrably slower build than a release build. For a release build,
# use "make -j`nproc` BUILD_TYPE=release"
```
and same in the OS X instructions below